docs/releases.md

```diff
@@ -22,7 +22,7 @@
 
 #### 4. Release kops to homebrew
 
-* Following the [documentation](development/homebrew.md) we must release a compatible homebrew formulae with the release.
```

Comment:
I think this is actually the correct spelling

https://formulae.brew.sh/

Reply:
Agreed on this.